### PR TITLE
chore: pushsync valid stamp metrics

### PR DESCRIPTION
**Pull request by istae**
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/2692)

### CODE REVIEW

**Review by acud — APPROVED** [review]

**Review by aloknerurkar — APPROVED** [review]

File: pkg/pushsync/metrics.go [hunk 1]
```diff
@@ -102,6 +103,12 @@ func newMetrics() metrics {
 			Name:      "invalid_stamps",
 			Help:      "No of invalid stamp errors.",
 		}),
+		StampValidationTime: *prometheus.NewGaugeVec(prometheus.GaugeOpts{
```
Comment (acud) [inline pkg/pushsync/metrics.go:106]: why not histogram?

Comment (istae) [reply pkg/pushsync/metrics.go:106]: validation duration should be more or less a static value but I don't mind changing it to a histogram to plot fluctuations if they do ever happen.

**Review by acud — APPROVED** [review]

File: .github/workflows/beekeeper.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
       REPLICA: 3
       RUN_TYPE: "PR RUN"
       SETUP_CONTRACT_IMAGE_TAG: "0.2.0"
-      BEEKEEPER_BRANCH: "master"
+      BEEKEEPER_BRANCH: "ps-lightnode-fix"
```
Comment (mrekucci) [inline .github/workflows/beekeeper.yml:17]: Shouldn't be this `master` branch?

Comment (istae) [reply .github/workflows/beekeeper.yml:17]: yep, just trying to fix a flaky test with that branch, it will be back to master when merging

**Review by mrekucci — APPROVED** [review]

**Pull request merged**